src/components/branding/branding.constant.ts

```diff
@@ -11,123 +11,62 @@
  */
 'use strict';
 
-export type IBranding = {
-  title: string,
-  name: string,
-  productVersion?: string;
-  logoFile: string,
-  logoTextFile: string,
-  favicon: string,
-  loader: string,
-  websocketContext: string,
-  helpPath: string,
-  helpTitle: string,
-  supportEmail: string,
-  oauthDocs: string,
-  cli: {
-    configName: string;
-    name: string;
-  },
-  docs: IBrandingDocs,
-  workspace: IBrandingWorkspace,
-  footer: IBrandingFooter,
-  configuration: IBrandingConfiguration,
-}
-
-export type IBrandingDocs = {
-  devfile: string,
-  workspace: string,
-  factory: string,
-  organization: string,
-  general: string,
-  converting: string,
-  certificate: string,
-  faq?: string,
-  storageTypes: string,
-  webSocketTroubleshooting: string,
-}
-
-export type IBrandingWorkspace = {
-  priorityStacks: Array<string>,
-  defaultStack: string
-}
-
-export type IBrandingFooter = {
-  content: string,
-  links: Array<{ title: string, location: string }>,
-  email: { title: string, address: string, subject: string } | null
-}
-
-export type IBrandingConfiguration = {
-  menu: {
-    disabled: che.ConfigurableMenuItem[];
-    enabled?: che.ConfigurableMenuItem[];
-  },
-  prefetch: {
-    cheCDN?: string;
-    resources: string[];
-  },
-  features: {
-    disabled: TogglableFeature[];
-  }
-}
-
-export enum TogglableFeature {
-  WORKSPACE_SHARING = 'workspaceSharing',
-  KUBERNETES_NAMESPACE_SELECTOR = 'kubernetesNamespaceSelector',
-}
+/**
+ * Generated content. Any changes will be lost after project build.
+ */
```

Comment:
We shouldn't commit this file. It is 'Generated content'.

Reply:
We've already have such files committed into repository, do you think we should treat only this file differently, or should we get rid of all generated files?

Reply:
I am sure we should rid of all files which are generated each build.

Reply:
I've just noticed that this makes the IDE complaining about wrong imports of those files. So I'm reverting the commit that removes this file.